about/migrations/0001_initial.py

```diff
@@ -18,10 +18,10 @@ class Migration(migrations.Migration):
                 ('picture', models.ImageField(default=None, null=True, upload_to=b'member-pics/', blank=True)),
                 ('first_name', models.CharField(max_length=100)),
                 ('last_name', models.CharField(max_length=100)),
-                ('program', models.CharField(default=b'Accounting and Financial Management', max_length=150)),
-                ('year', models.CharField(default=b'2019', max_length=100)),
+                ('program', models.CharField(max_length=100)),
+                ('year', models.CharField(max_length=100)),
```

Comment:
This would fail on new databases; don't ever directly edit migrations. Instead, edit the model and have Django generate a new one for you.